front/app/components/UI/MultipleSelect/styles.ts

```diff
@@ -40,6 +47,10 @@ export function getSelectStyles(borderColor = colors.borderDark) {
       ...base,
       color: '#999',
     }),
+    menuList: (base) => ({
+      ...base,
+      borderRadius: '3px',
```

Comment:
Maybe better to use the `stylingConsts` `borderRadius`? It's the same value but comes from the component library, meaning if we ever decide to switch it up, it would be easier.